roofit/roofitcore/src/RooAbsPdf.cxx

```diff
@@ -205,6 +205,35 @@ bool interpretExtendedCmdArg(RooAbsPdf const& pdf, int extendedCmdArg) {
   return extendedCmdArg;
 }
 
+inline double getLog(double prob, RooAbsReal const *caller)
```

Comment:
Is this  a public function ? 
I think it should be protected and maybe having a more informative name

Reply:
This is only in the `.cxx` file in the anonymous namespace, so not public at all